src/hotspot/share/memory/metaspace/freeBlocks.cpp

```diff
@@ -53,7 +50,7 @@ MetaWord* FreeBlocks::remove_block(size_t requested_word_size) {
     // Blocks which are larger than a certain threshold are split and
     //  the remainder is handed back to the manager.
     const size_t waste = real_size - requested_word_size;
-    if (waste > MinWordSize) {
```

Comment:
Has the calculation been wrong before or should that be > 1 or am I missing something?

Reply:
The calculation before had been wrong; MinWordSize is inclusive